rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -509,6 +516,51 @@ void logPegoutConfirmed() {
         assertEvent(eventLogs, 0, BridgeEvents.PEGOUT_CONFIRMED.getEvent(), new Object[]{btcTx.getHash().getBytes()}, new Object[]{pegoutCreationRskBlockNumber});
     }
 
+    private static Stream<Arguments> logPegoutTransactionCreatedValidArgProvider() {
+        List<Arguments> args = new ArrayList<>();
+
+        args.add(Arguments.of(Sha256Hash.ZERO_HASH, Collections.singletonList(Coin.SATOSHI)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(5), coinListOf(500_000, 400_000, 1_000)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(10), Collections.singletonList(Coin.ZERO)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(10), Collections.EMPTY_LIST));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(15), null));
```

Comment:
Shouldn't this one throw an exception?

Reply:
No, it shouldn't throw any exception due to the current implementation of the [encoding util method](https://github.com/rsksmart/rskj/blob/hsm-segwit-compatibility-integration/rskj-core/src/main/java/co/rsk/peg/bitcoin/UtxoUtils.java#L64). When passing null, it returns an empty array as encoded value for nulls.

Do you think this should be changed to not allow either null or empty list?